From 715e33eca355cb3a44900eeb50c8d2d1deeaf6a7 Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Wed, 20 Mar 2019 14:53:51 +0800
Subject: [PATCH 611/706] vfio fsl-mc: remove this insave iormb since readq
 already has it.

According to the definition of readq in arch/arm64/include/asm/io.h
__iormb has already been appended, so we can totally remove it.

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/vfio/fsl-mc/vfio_fsl_mc.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/vfio/fsl-mc/vfio_fsl_mc.c b/drivers/vfio/fsl-mc/vfio_fsl_mc.c
index 9d134529f447..019d8fc2cb92 100644
--- a/drivers/vfio/fsl-mc/vfio_fsl_mc.c
+++ b/drivers/vfio/fsl-mc/vfio_fsl_mc.c
@@ -331,9 +331,7 @@ static int vfio_fsl_mc_dprc_wait_for_response(void __iomem *ioaddr)
 		u64 header;
 		struct mc_cmd_header *resp_hdr;
 
-		__iormb();
 		header = readq(ioaddr);
-		__iormb();
 
 		resp_hdr = (struct mc_cmd_header *)&header;
 		status = (enum mc_cmd_status)resp_hdr->status;
-- 
2.17.1

